lib/sqlalchemy/sql/operators.py

```diff
@@ -2195,6 +2255,12 @@ class _OpLimit(IntEnum):
     neg: 8,
     add: 7,
     sub: 7,
+    bitwise_xor_op: 7,
+    bitwise_or_op: 7,
+    bitwise_and_op: 7,
+    bitwise_not_op: 7,
+    bitwise_lshift_op: 7,
+    bitwise_rshift_op: 7,
```

Comment:
I am sooo not sure if that is the right precedence. Looks close enough, but I think `~`, at the very least, should have precedence over other bitwise operators.